src/components/InpuField.css

```
@@ -1,21 +1,50 @@
-.inputfield {
+.Add,
```

class names should be of lowercase

```suggestion
.add
```